frontend/src/lib/components/details/seasons.svelte

```diff
@@ -1,124 +1,64 @@
-                <div class="text-xl text-netural-content">
-                  {season.air_date ? season.air_date.split("-")[0] : "No air date"}
-                  | {season.episode_count} episodes
-                </div>
-                <div class="text-lg text-netural-content">
-                  {season.air_date ? `Premiered on ${season.air_date}` : "Hasn't aired"}
-                </div>
-                &nbsp
-                <div class="text-netural-content">
-                  {season.overview ? season.overview : "No season overview available."}
-                </div>
-              </div>
-            </div>
-          {/if}
-        {/if}
-      </MediaQuery>
-      <MediaQuery query="(min-width: 1025px)" let:matches>
-        {#if matches}
-          {#if index === currentTab}
-            <div class="card card-side bg-neutral bordered lg:w-5/6 xl:w-3/5">
-              {#if season.poster_path}
-                <figure>
-                  <img
-                    src="{IMG_W342}{season.poster_path}"
-                    class="object-fit h-96"
-                    alt={season.name}
-                  />
-                </figure>
-              {:else}
-                <figure>
-                  <img
-                    src="../../no_image_available.jpg"
-                    class="object-fit h-72"
-                    alt="No poster available"
-                  />
-                </figure>
-              {/if}
-              <div class="mx-4 my-2">
-                <div class="card-title">{season.name}</div>
-                <div class="text-xl text-netural-content">
-                  {season.air_date ? season.air_date.split("-")[0] : "No air date"}
-                  | {season.episode_count} episodes
-                </div>
-                <div class="text-lg text-netural-content">
-                  {season.air_date ? `Premiered on ${season.air_date}` : "Hasn't aired"}
-                </div>
-                &nbsp
-                <ReadMore
-                  currentDescriptionLength={currentOverviewLength}
-                  mediaDescription={season.overview
-                    ? season.overview
-                    : "No season overview available."}
-                  initialDescriptionLength={INITIAL_OVERVIEW_LENGTH}
+  <div class="swiper-container cursor-pointer">
+    <Swiper
+      style="
+              --swiper-navigation-color: hsl(var(--sc));
+              --swiper-navigation-size: 25px;
+            "
+      breakpoints={{
+        576: { slidesPerView: 1.6, spaceBetween: 10 },
+        768: { slidesPerView: 2, spaceBetween: 10 },
+        992: { slidesPerView: 2.8, spaceBetween: 10 },
+        1280: { slidesPerView: 3.2, spaceBetween: 10 },
+        1400: { slidesPerView: 3.8, spaceBetween: 10 },
+      }}
+      navigation={true}
+      modules={[Navigation]}
+      initialSlide={seasons[0].name === "Specials" ? 1 : 0}
+    >
+      {#each seasons as season, index}
+        <SwiperSlide>
```

Comment:
Not sure if it's possible or not, but it would be nice to lazy load this(20+ seasons isn't super uncommon)

Reply:
Done